Vagrantfile

```diff
@@ -0,0 +1,72 @@
+# -*- mode: ruby -*-
```

Comment:
so, just need to remove this now, yeah?

Reply:
yep. removed this and `vm-setup.sh`.

Reply:
Can't delete `vm-setup.sh`. Travis needs it.